pkg/reconciler/pingsource/pingsource.go

```diff
@@ -111,10 +111,6 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, source *v1alpha2.PingSou
 	}
 	source.Status.MarkSink(sinkURI)
 
-	// The webhook does not allow for invalid schedules to be posted.
-	// TODO: remove MarkSchedule
-	source.Status.MarkSchedule()
```

Comment:
Why is this deleted? I don't have a lot of context, sorry if I am asking something silly.

Reply:
@aliok For pingsource v1beta1 lifecycle, the PingSourceConditionValidSchedule 
 condition was removed, along with others (https://github.com/knative/eventing/pull/3607). MarkSchedule() in v1alpha2 also has a comment `// TODO: this is a bad method name, change it.` . Don't know what the plan is here cc: @lionelvillard

Reply:
yes this is a good change! The schedule is validated by the webhook so no need for a condition.

Reply:
All good, thanks